index.html

```diff
@@ -9,6 +9,26 @@
 </head>
 
 <body>
+    <div class="top-header">
+        <ul class="contact-cotainer">
+            <li class="contact-item">
+                <a href="" class="facebook"> <img src="assets/images/facebook.svg" alt=""> </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="twitter"> <img src="assets/images/twitter.svg" alt=""> </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="language"> Lanuage </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="mypag"> My Page </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="logout"> logout </a>
+            </li>
+          
+        </ul>
```

Comment:
- [ ] I must commend your design skills, your page looks really nice, and thanks to the previous reviewer, your looks better now... :clap:  The only issue is that your additional nav bar is causing a design conflict that is currently not too good for user experience. How about hiding it completely when the screen width is `< 768px`? Or better still, you can add it to the main menu in your mobile view :wink: 

![image](https://github.com/akbarsazish/Sazish-Academy/assets/99715640/09bb5b7e-1683-4e0c-9036-81a509fa40c2)

- [ ] [OPTIONAL] Your hamburger menu doesn't have a close option to collapse the menu should the user decide not to click on any button. I think this is a great feature that can further increase user experience :+1: 
![image](https://github.com/akbarsazish/Sazish-Academy/assets/99715640/e1bb858f-f105-4a40-a62e-faf1bc62245b)

Reply:
Hi @AbayomiOlaoye, In my computer I hide the nave bar (top nav) and also mobile menu has the close button, everythings is working pefeclty. Please press ctrl + f5 on computer or somthing else. 
As I don't have the cases that mentioned I don't know what to do, as you know it is the capstone project I have to outline the presentation and record the vidoe so, my time is limited. 

please let me know what should I do.

![top nav is hide](https://github.com/akbarsazish/Sazish-Academy/assets/97288972/14f77fbd-e0aa-4eef-a3d3-d652215c05e1)
![close btn](https://github.com/akbarsazish/Sazish-Academy/assets/97288972/4816784c-349f-4cd1-bf11-2d2a39fe07e5)

Reply:
Should I submit the pull request again or ???

Reply:
.top-header,
.hide-btn {
  display: none;
}

the above code hide that menu while it is  > 768px

Reply:
@akbarsazish, Now, that's a real issue. It does meant that it has worked before. Kindly clear your browser's cache memory or try it on another browser. If everything checks well, you can submit again

Reply:
@AbayomiOlaoye, I have cleared the browser cache and checked with two different browser it works.